marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java

```diff
@@ -659,4 +659,8 @@ public boolean savePii() {
         return true;
     }
 
+    public void setHubConfig(HubConfig hubConfig) {
```

Comment:
Same thing here, let's use a constructor here instead of a setter so it's clear that a new object is being created that's specific to this HubConfig.